src/test/java/integration/EsContainer.java

```diff
@@ -0,0 +1,23 @@
+package integration;
+
+
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class EsContainer extends ElasticsearchContainer {
+
+    public EsContainer() {
+        super(DockerImageName.parse("elasticsearch:8.5.3")
```

Comment:
@hizmailovich try stay away from code inside the ctor

Reply:
Updated